formats/log/README.md

```diff
@@ -45,25 +45,42 @@ signatures, the format itself is not restricted to this scheme.
 The checkpoint body is of the form:
 
 ```text
-<Ecosystem & version string>
+<Origin string>
 <Decimal log size>
 <Base64 log root hash>
 [otherdata]
 ```
 
 The first 3 lines of the body **MUST** be present in all Checkpoints.
-`otherdata` is opaque and optional, and, if necessary, can be used to tie extra
-data to the checkpoint, however its format must conform to the sumdb signed
-note spec (e.g. it must not contain blank lines.)
+
+* `<Origin string>` should be a unique identifier for the log identity which issued the checkpoint.
+  The exact format of this is left undefined, but examples of types of value to put here
+  are the log's URL, a base64 encoded hash of its public key, etc.
+
+  The presence of this identifier forms part of the log claim, and prevents witness
+  signatures being moved between log checkpoints with identical contents, preventing a
+  particular type of targetted attack.
+
+* `<Decimal log size>` is the ASCII decimal representation of the number of leaves committed
+  to by this checkpoint. It should not have leading zeroes.
+
+* `<Base64 log root hash>` is an
+  [RFC4684 standard encoding](https://datatracker.ietf.org/doc/html/rfc4648#section-4) base-64
+  representation of the log root hash at the specified log size.
+
+* `[otherdata]` is opaque and optional, and, if necessary, can be used to tie extra
+  data to the checkpoint, however its format must conform to the sumdb signed
+  note spec (e.g. it must not contain blank lines.)
 
 > Note that golang sumdb implementation is already compatible with this
 `[otherdata]` extension (see
 <https://github.com/golang/mod/blob/d6ab96f2441f9631f81862375ef66782fc4a9c12/sumdb/tlog/note.go#L52>).
 
-The first signature on a checkpoint must be from the log which issued it.
+The first signature on a checkpoint should be from the log which issued it, but there MUST NOT
+be more than one signature from a log identity present on the checkpoint ()
```

Comment:
Not sure what's happened with the empty brackets?

Reply:
I think I was going to add a parenthetical about it being nonsensical to have a signature on a checkpoint from a log other than the one identified by the zeroth line, but that's probably not really necessary so I've ding'd 'em.